packages/odyssey-react/src/components/Tabs/Tabs.module.scss

```
@@ -40,15 +42,14 @@
 }
 
 /* Tab states */
-
-.selectedState {
+.selected {
   &::before {
     content: "";
     position: absolute;
     inset-block-end: -1px;
     inset-inline-start: 0;
     width: 100%;
     height: 3px;
-    background: $color-primary-base;
+    background: var(--BackgroundColorSelected);
```

```suggestion
    background-color: var(--ColorBackgroundSelected);
```

let's avoid the shorthand declaration here and use a more specific `background-color` declaration. We also group background colors under the `ColorBackground` prefix with other colors.